plugins/inputs/cisco_telemetry_mdt/README.md

```diff
@@ -0,0 +1,51 @@
+# Cisco model-driven telemetry (MDT)
+
+Cisco model-driven telemetry (MDT) is an input plugin that consumes
+telemetry data from Cisco IOS XR, IOS XE and NX-OS platforms. It supports TCP & GRPC dialout (server) and GRPC dialin (client) transports.
+GRPC-based transport can utilize TLS for authentication and encryption.
+Telemetry data is expected to be GPB-KV (self-describing-gpb) encoded.
+
+The GRPC dialout transport is supported on various IOS XR (64-bit) 6.1.x and later, IOS XE 16.10 and later, as well as NX-OS 7.x and later platforms.
+
+The GRPC dialin transport is supported on IOS XR (64-bit) 6.1.x and later.
+
+The TCP dialout transport is supported on IOS XR (32-bit and 64-bit) 6.1.x and later.
```

Comment:
I removed the dial-in for now, since it is of lesser importance for now. I will talk to my colleagues and see if that is fine for now or if we add this again add a later stage.

As for the other comments I tried to apply similar changes I have made to GNMI based on your comments to this plugin as well.